Packs/Jira/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Atlassian Jira",
     "description": "Use the Jira integration to manage issues and create Cortex XSOAR incidents from Jira projects.",
     "support": "xsoar",
-    "currentVersion": "3.1.2",
+    "currentVersion": "3.2.0",
```

Comment:
@omerKarkKatz Is adding a new command considered a minor or revision?

Reply:
This is more fit for a revision update, I will modify the version and release notes in the inner PR.

Reply:
:+1: